tests/test_templates.py

```diff
@@ -72,7 +72,7 @@ def test_load_samples():
     cwd = os.getcwd()
     os.chdir(FIXTURES)
 
-    common_templates = common.CommonTemplates()
+    common_templates = gcp.CommonTemplates()
```

Comment:
not a blocker at all, but I might have been tempted to make this change in a separate PR. This is just a better approach to importing, and shouldn't have an effect on the functionality?

Reply:
I will revert this and move it to a separate PR.